.travis.yml

```diff
@@ -7,36 +8,38 @@ notifications:
     on_cancel: always
     rooms:
       secure: uFHb/4trdoMHdeO1kTkY+lBh+pU70CW69mBLnBv8Z6IclNkh8C5fA+/BvzhRMAanZbT+eSIaZE2lb7soLm0J4aXsG1xyEMhAzsa7UgbzXiWYc8suhE2Xf2mR1jR4YyVqFc4P+3WKhIEQdDSTkt/L3Y7unCE6NsipF2IUmUoFAjZO6Ykr1kXkw1y1DGeV8abI4ET5DTCe/KWazQMnePqShndRpPR70YOze0NzzJBqb4zRnC8G0qMp7c04G7byxi5VLC+IqcKcQIXT3TowlWGF8V3nruNcihZVjU5TXOEvedJMXWdqiif6jhymBD4rUh8pQeYuXCGEdUq78DgcbswtkbDgT5P8Jz85dF432prMgXvsnRb0EqKE+NFmkwTytpa6C/07fkbP0z08WaHu4BUb+JCLcao4c0sB1MhrQWqChDT+fSErZ1PqubXzw+PVTZMo4inONwrbiDMQnmJnJXLxOW+LWuUQh3r6Ue7Vry2wNVLL05VoRAZdXOiCvs0cfo4VImcSkgTaUTlrVwdF8nnaCB9TlqGqQcGEZDY0Fm+ZSD30fSu8Oy0JV04+F3rW2GQLbOlHi3X2FOJKoBXi6hQ5U6DkOkC65V/Vij7q0x508pi0wjDcTcDrQq9XunLBZz6zSuMG1VGuWmvRKLYhdGkIoA47zrfHMq1kzjL5AWxfbPs=
+
 services:
-  - mongodb
-  - rabbitmq
   - docker
+
 env:
   - REPO=poetapp/node
-before_install:
-  - sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
-  - sudo apt-get update
-  - sudo apt-get install gcc-5 g++-5
-  - sudo rm /usr/bin/g++
-  - sudo ln -s /usr/bin/g++-5 /usr/bin/g++
-before_script:
-  - sudo make ipfs
-  - npm run build
-  - npm run test-prepare-db
-  - sleep 5
-  - npm start &
-  - sleep 5
+
+install:
+  - DC_FILES="-f docker-compose.yml -f docker-compose.ci.yml"
+  - docker-compose $DC_FILES build
+  - docker images
+  - docker-compose $DC_FILES up -d
```

Comment:
I would have thought this would stop the logs from showing up in the Travis report, but looks like not... any idea why?